src/getting_started/platform-specific_setup.md

```diff
@@ -54,6 +54,67 @@ Depending on what OS you are running, you might require an extra step or two.
     [others](https://en.wikipedia.org/wiki/LibreSSL#Adoption) if manually
     installed).
 
+  - **vulkan**
+
```

Comment:
Perhaps it's worth adding the following text here before guiding users to the commands below:

> Installing Vulkan support on Linux is generally quite easy using your distro's package manager. That said, there may be different driver options to consider depending on your graphics card and tolerance for proprietary software. The following are rough guidelines on how to get going quickly, however if you are at all concerned with finding the approach that suits you best we recommend searching for vulkan driver installation on your distro.

cc @freesig would this address your concerns re the possibility for different drivers? I honestly don't think we'll ever be able to feasibly automate Linux installation entirely due to the number of everchanging distros, but I also think Linux users are generally used to adding a couple of commands like this anyways. Maybe we can revisit the possibility of automating it on at least some distros if we have a go at a graphical installer in the future?

Reply:
added the text